### PR TITLE
Fixed grammatical errors on Term page

### LINKED ISSUE #2013: Fix grammatical error on terms page.
**Actual Behaviour**

Grammatical and punctuation errors are there on terms page. 

**Expected Behaviour**

Remove the errors.

**Would you like to work on the issue?**

Yes, I would like to work on this.

### PR DESCRIPTION
**Pull request by plxity**
Fixes #2013 

Changes: Fixed grammatical errors on the term page.

Demo Link: https://pr-2015-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/Terms/Terms.react.js [hunk 1]
```diff
@@ -282,8 +282,9 @@ class Terms extends Component {
                 indemnify SUSI and its affiliates, officers, agents, and
                 employees from any claim, suit or action arising from or related
                 to the use of the Services or violation of these terms,
-                including any liability or expense arising from claims, losses ,
-                damages, suits, judgments, litigation costs and attorneys’ fees.
+                including any liability or expense arising from claims, losses,
+                damages, suits, judgments, litigation costs, and attorneys’
```
Comment (AakashMallik) [inline src/components/Terms/Terms.react.js]: costs and attorneys’

Comment (anshumanv) [reply src/components/Terms/Terms.react.js]: It's okay I guess - 

https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

@AakashMallik can you proofread the whole file to see if we're not missing anything just to be sure? 😄

Comment (plxity) [reply src/components/Terms/Terms.react.js]: Yeah, it would be nice. I checked it completely. In case I missed something let me know.

Comment (AakashMallik) [reply src/components/Terms/Terms.react.js]: @anshumanv Sure I'll give it a read ASAP

Comment (plxity) [reply src/components/Terms/Terms.react.js]: @AakashMallik I hope it is fine?

Comment (AakashMallik) [reply src/components/Terms/Terms.react.js]: @plxity I'll give it a read at around 22:00. Sorry for the delay but I'm a little caught up in something.

**Comment by plxity** [comment]
@AakashMallik  Okay bro 👍

**Comment by plxity** [comment]
@anshumanv I think this can be merged now. :)

**Comment by plxity** [comment]
@anshumanv Can this be merged?

**Pull request merged**